odoo/http.py

```diff
@@ -2258,6 +2258,17 @@ class Application:
     """ Odoo WSGI application """
     # See also: https://www.python.org/dev/peps/pep-3333
 
+    def initialize(self):
+        """
+        Initialize the application.
+
+        This is to be called when setting up a WSGI application after
+        initializing the configuration values.
+        """
+        module_manager.initialize_sys_path()
+        from odoo.service.server import load_server_wide_modules  # noqa: PLC0415
+        load_server_wide_modules()
+
```

Comment:
Make sense to have it on the Application class, but I'm not too sure it is better than to call those two methods in the example wsgi file

Reply:
The goal is that each caller should not think about what to call.